doc/lua_api.md

```diff
@@ -6922,6 +6922,10 @@ Timing
     * Optional: Variable number of arguments that are passed to `func`
     * Jobs set for earlier times are executed earlier. If multiple jobs expire
       at exactly the same time, then they are executed in registration order.
+    * `time` is a lower bound. The job is executed in the first server-step that
+      started at least `time` seconds after the last time a server-step started,
+      measured with globalstep dtime.
```

Comment:
something like "In particular this can result in relatively large delays if `time` is close to the server step time."?

Reply:
Pushed something. Comments?